.github/workflows/ci.yml

```diff
@@ -122,6 +122,7 @@ jobs:
             echo "::set-output name=matrix::{\"images\": \
                                                 [{\"image\": \"debian\", \"marker\": \"ingresses\"}, \
                                                 {\"image\": \"alpine\", \"marker\":\"vsr\"}, \
+                                                {\"image\": \"alpine-opentracing\", \"marker\":\"ingresses\"}, \
```

Comment:
If you change this from "ingresses" to "policies", you can remove the "debian-policies" test below (we now have 5 images and 5 test markers so can do a 1-2-1 for each)

Reply:
done